docs/cep/table/index.md

```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
 | attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
-| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
-| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      |
+| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. |
```

Comment:
@diana-macrometa - The both parameters could be set only with `STORE` and should be move to the table below. 
Other store parameters are:
- `type`, e.g. type = 'database'
- `collection`, e.g. collection = "SampleCargoAppInputTable", 
- `collection.type`, e.g. collection.type="doc" , 
- `replication.type`, e.g. replication.type="global", 
- `map.type`, e.g. map.type='json'